source/utilities.tex

```diff
@@ -2047,7 +2040,7 @@
 \indexlibrary{\idxcode{tuple_cat}}
 \begin{itemdecl}
 template <class... Tuples>
-  constexpr tuple<@\placeholder{CTypes}@...> tuple_cat(Tuples&&... tpls);
+  constexpr tuple<CTypes...> tuple_cat(Tuples&&... tpls);
 \end{itemdecl}
 
 \begin{itemdescr}
```

Comment:
@zygoloid: I hadn't revised `tuple_cat` at all, and that was still the original formatting. I've now added a complete review of this description to the PR.